docs/root/intro/arch_overview/listeners/listeners.rst

```diff
@@ -5,17 +5,20 @@ Listeners
 
 The Envoy configuration supports any number of listeners within a single process. Generally we
 recommend running a single Envoy per machine regardless of the number of configured listeners. This
-allows for easier operation and a single source of statistics. Currently Envoy only supports TCP
+allows for easier operation and a single source of statistics. Envoy supports both TCP and UDP
```

Comment:
Can we proxy UDP yet?  If not I'd be inclined to hide and say it's not supported.  Your call.

Reply:
We can't proxy UDP yet (I will be working on this soon), but we do support UDP listeners and filters so this seems OK?

Reply:
Accurate but I guess IMO confusing as we support listeners but they "don't work" unless you want to echo.

Reply:
OK I can remove for now.